packages/flutter_driver/lib/src/extension/extension.dart

```diff
@@ -166,91 +234,84 @@ abstract class FinderExtension {
   Finder createFinder(SerializableFinder finder, CreateFinderFactory finderFactory);
 }
 
+/// Used to expand the new Command
+abstract class CommandExtension {
+
+  /// Identifies the type of command to be used by the driver extension.
+  String get commandKind;
+
+  /// Deserializes the command from JSON generated by [Command.serialize].
+  /// [finderFactory] could be used to deserialize nested finders.
+  /// [commandFactory] could be used to deserialize nested commands.
```

Comment:
Documentation should be in complete sentences that begin with a capital letter and end with a period.

We should also avoid words like "should" or "could" in documentation and instead say what actually must be the case for different conditions.

Reply:
Corrected. As well as previous PRs entries.